app/routes/index.js

```diff
@@ -31,6 +33,15 @@ module.exports = function init(app) {
   app.get('/', statusRoute);
   app.use('/v1/status', statusRoute);
 
+  // Provides data for a chatbot broadcast.
+  app.use('/v1/broadcasts/:broadcastId', broadcastsSingleRoute);
```

Comment:
Just a thought... maybe we can start the move towards `/v2` at this time with these "simpler" routes?

Reply:
I'm leaning more towards `v1` as our `defaultTopicTriggers` and `topics` are defined there as well. How about holding off on v2 until we split out the `broadcast` type into different content types, and then could potentially make the change across all endpoints?